src/azure/azureController.ts

```diff
@@ -211,4 +207,19 @@ export class AzureController {
         await azureAuth.deleteAccountCache(account.key);
         return;
     }
+
+    /**
+     * Checks if this.init() has already been called
```

Comment:
Would suggest actually saying what this is doing (like what does it "initialize") and mention that it will only do that stuff once.